Emu/.emu_setup/add_favourite.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+. /mnt/SDCARD/spruce/scripts/helperFunctions.sh
+
+##### DEFINE BASE VARIABLES #####
+
+FAVOURITE_FILE="/mnt/SDCARD/Roms/favourite.json"
+EMU_NAME="$(echo "$1" | cut -d'/' -f5)"
+ROMFILENAME="$(basename "$1")"
+GAMENAME="$(echo "$ROMFILENAME" | cut -d'.' -f1)"
+
+log_message "--- Adding favorite game chosen by user ---"
+
+# Construct the new JSON entry
+NEW_ENTRY=$(jq -n \
+  --arg label "$GAMENAME ($EMU_NAME)" \
+  --arg launch "/mnt/SDCARD/Emu/$EMU_NAME/launch.sh" \
```

Comment:
We have removed individual emu launch scripts in favour of directly running `.emu_setup/stanard_launch.sh`